alf/utils/common.py

```diff
@@ -1440,11 +1440,34 @@ def rsync(src, target, includes):
     if alf_dirname != "alf":
         os.system("mv %s/%s %s/alf" % (dest_path, alf_dirname, dest_path))
 
+    # compress the snapshot repo into a ".tar" file
+    os.system("cd %s; tar -czf alf.tar alf" % dest_path)
+    os.system("rm -rf %s/alf" % dest_path)
+
+
+def unzip_alf_snapshot(root_dir: str):
+    """Restore an ALF snapshot from a job directory by unzipping the snapshot
+    'tar' file.
+
+    Args:
+        root_dir: the tensorboard job directory
+    """
+    alf_zipped_repo = os.path.join(root_dir, "alf.tar")
```

Comment:
Check for the existence of `tar` so that we can prompt clear information when it does not?

Reply:
> Check for the existence of `tar` so that we can prompt clear information when it does not?

Already checked below?